todoapp/app/src/main/res/values/colors.xml

```diff
@@ -23,4 +23,6 @@
 
     <drawable name="touchFeedback">#CFD8DC</drawable>
 
+    <drawable name="selectedTask">#CFD8DC</drawable>
```

Comment:
no need to duplicate this, how about calling a color `touchStateBackground` or similar and merge `touchFeedback` and `selectedTask`?